blocks/library/table/index.js

```diff
@@ -69,22 +90,40 @@ export const settings = {
 						onChange={ updateAlignment }
 					/>
 				</BlockControls>
+				<InspectorControls>
+					<PanelBody title={ __( 'Table Settings' ) } className="blocks-table-settings">
+						<ToggleControl
+							label={ __( 'Fixed width table cells' ) }
+							checked={ !! hasFixedLayout }
+							onChange={ toggleFixedLayout }
+						/>
+					</PanelBody>
+				</InspectorControls>
 				<TableBlock
 					onChange={ ( nextContent ) => {
 						setAttributes( { content: nextContent } );
 					} }
 					content={ content }
-					className={ className }
+					className={ classes }
 					isSelected={ isSelected }
 				/>
 			</Fragment>
 		);
 	},
 
-	save( { attributes } ) {
-		const { content, align } = attributes;
+	save( { attributes, className } ) {
+		const { content, align, hasFixedLayout } = attributes;
+
+		const classes = classnames(
+			className,
+			{
+				'has-fixed-layout': hasFixedLayout,
+			},
+			align ? `align${ align }` : null,
```

Comment:
`align ? `align${ align }` : null,` can be moved to the object as follows:
```js
{
	'has-fixed-layout': hasFixedLayout,
	[ `align${ align }` ]: align,
},
```

Reply:
Nice 👍 👍